### PR TITLE
Map view visual redesign

### PR DESCRIPTION
**Pull request by tomasMizera**
Updated map view to the new design:
 - toolbars (recording, splitting)
 - stakeout
 - banners
 - map buttons

| Some screenshots | |
|--------|--------|
|Screenshot 2024-01-19 at 09 08 17 |Screenshot 2024-01-19 at 09 09 30 |
|Screenshot 2024-01-19 at 09 09 35 |Screenshot 2024-01-19 at 09 10 35 |

Highlight will be added in another PR
Best to go over commits one by one :)

### CODE REVIEW

File: gallery/helper.cpp [hunk 1]
```diff
@@ -56,7 +56,6 @@ qreal Helper::calculateScreenDpr()
 
 qreal Helper::calculateDpRatio()
 {
-#if MOBILE_OS
```
Comment (PeterPetrik) [inline gallery/helper.cpp:59]: why removed?

**Review by PeterPetrik — COMMENTED** [review]
looks good

File: app/appsettings.h [hunk 2]
```diff
@@ -131,8 +126,6 @@ class AppSettings: public QObject
     bool mAutoCenterMapChecked = false;
     // used in GPS signal indicator
     double mGpsAccuracyTolerance = -1;
-    // flag for showing accuracy warning if is above tolerance
-    bool mGpsAccuracyWarning = true;
```
[the remark this answers is not in the document]
Comment (tomasMizera) [reply app/appsettings.h:135]: The GPS accuracy button serves that purpose now

File: app/inputhelp.cpp [hunk 3]
```diff
@@ -142,7 +142,7 @@ QString InputHelp::howToSetupProj() const
 
 QString InputHelp::gpsAccuracyHelpLink() const
 {
-  return helpRoot + "/field/gps_accuracy/" + utmTagHelp;
+  return helpRoot + "/field/gps_accuracy/" + utmTagHelp; // NOTE: keeping here for future use
```
[the remark this answers is not in the document]
Comment (tomasMizera) [reply app/inputhelp.cpp:145]: It will be used in GPS data page

File: gallery/helper.cpp (hunk 1 above)
Comment (tomasMizera) [reply gallery/helper.cpp:59]: To unify it with Input code. In gallery, `dp` was always set to 1, which made components look differently when used in Input vs gallery

File: app/notificationmodel.h [hunk 4]
```diff
@@ -87,8 +87,18 @@ class NotificationModel : public QAbstractListModel
 
     Q_PROPERTY( int rowCount READ rowCount NOTIFY rowCountChanged );
     Q_INVOKABLE void remove( uint id );
+
+    //! Adds a new notification with message, interval (in seconds), type and icon
     Q_INVOKABLE void add( const QString &message, uint interval, NotificationType::MessageType type, NotificationType::IconType icon );
 
+    //! Convinient methods to save typing of some parameters
+    Q_INVOKABLE void addSuccess( const QString &message );
```
[the remark this answers is not in the document]
Comment (tomasMizera) [reply app/notificationmodel.h:95]: I prefer not going down that path, we can instead define additional methods also with expiration. QML/JS engine does not work well with C++'s default arguments or method overloads.

File: app/qml/StakeoutPanel.qml [hunk 5]
```diff
@@ -262,7 +262,7 @@ Item {
             },
             State {
               name: "notAtTarget"
-              when: state != "atTarget"
+              when: state !== "atTarget"
```
[the remark this answers is not in the document]
Comment (tomasMizera) [reply app/qml/StakeoutPanel.qml:265]: Not necessarily, it was JS warning. This way it won't perform type coercion https://medium.com/@atuljha2402/understanding-javascript-type-coercion-type-conversion-a2ce84c00331#:~:text=Comparison%20Coercion%3A

File: app/qml/map/Crosshair.qml [hunk 6]
```diff
@@ -103,7 +103,7 @@ Item {
     ColorOverlay {
       anchors.fill: crosshairForeground
       source: crosshairForeground
-      color: snapUtils.snapped ? "#8a2be2" : InputStyle.fontColor
+      color: snapUtils.snapped ? "#8a2be2" : __style.forestColor // grapeColor replaced previous "#8a2be2" used for snapping
```
[the remark this answers is not in the document]
Comment (tomasMizera) [reply app/qml/map/Crosshair.qml:106]: I am waiting for Andrej to define color for snapping. Tried grape previously, but it did not work well.

File: app/qml/components/MMToolbarButton.qml [hunk 7]
```diff
@@ -10,6 +10,7 @@
 import QtQuick
 import QtQuick.Controls
 import QtQuick.Controls.Basic
+import Qt5Compat.GraphicalEffects
```
[the remark this answers is not in the document]
Comment (tomasMizera) [reply app/qml/components/MMToolbarButton.qml]: Not anymore, good catch

**Review by PeterPetrik — APPROVED** [review]

**Pull request merged**